pylhc/autosix.py

```diff
@@ -0,0 +1,525 @@
+"""
+AutoSix
+-------
+
+A wrapper for SixDesk to perform the setup and steps needed automatically.
+
+The idea is, to call this script with the same parameters and it runs through
+all the steps automatically.
+
+The functionality is similar to the :mod:`pylhc.job_submitter` in that
+the inner product of a ``replace_dict`` is used to automatically create a set of
+job-directories to gather the data.
+To avoid conflicts each of these job-directories is a SixDesk workspace,
+meaning there will be only one "study" per directory, .
+
+The ``replace_dict`` contains variables for your mask as well as variables
+for the SixDesk environment. See the description of ``replace_dict`` below.
+In any other way, these __special__ variables behave like normal variables and
+can also be inserted in your mask. They are also looped over in the same manner
+as any other variable (if given as a list).
+
+.. important::
+    As the loop over Seeds is handled by SixDesk you need to set
+    `FIRSTSEED` and `LASTSEED` to ``None`` or ``0`` to deactivate this loop.
+    Otherwise a ``%SEEDRAN`` placeholder is required in your mask,
+    which needs to be present **after** filling in the variables (see example below).
+
+
+.. note::
+    Unlike in the :mod:`pylhc.job_submitter`, the output directory of the
+    HTCondor job (the 'mask-job') is not automatically transferred to the
+    workspace. To have access to this data, you will need to specify different
+    output-directories in your mask manually, e.g. using strings containing
+    the variable placeholders.
+
+
+.. code-block:: python
+
+    from pathlib import Path
+    from omc3.utils import logging_tools
```

Comment:
I don't think the `ready-to-run` example should require `omc3`, this could easily be `import logging`

Reply:
it's pylhc. everything requires omc3. 
The import of autosix requires omc3.
also, otherwise you have to setup logger manually (import logging does not create any output).

Reply:
The following works:
```python
import logging
logging.warning("message")
```
But ok.